app/helpers/actors_helper.rb

```diff
@@ -0,0 +1,9 @@
+module ActorsHelper
+  def setActor(actor)
```

Comment:
name should explanatory:
you are fetching actors url.
and the method name is also not correct.